lib/usd/hdMaya/adapters/materialAdapter.cpp

```diff
@@ -666,6 +659,10 @@ class HdMayaShadingEngineAdapter : public HdMayaMaterialAdapter {
 
         HdMaterialNetworkMap materialNetworkMap;
         materialNetworkMap.map[HdMaterialTerminalTokens->surface] = materialNetwork;
+#if USD_VERSION_NUM >= 1911
+        materialNetworkMap.terminals.push_back(
+            materialNetwork.nodes.back().path);
```

Comment:
Do we know that ```materialNetwork.nodes``` will never be empty here?

Reply:
I think as it's currently written, if `GetMaterial()` above returns success, we *must* have added nodes to the network. It's a good idea to be a little paranoid though in case that changes at some point. Good catch!

Reply:
Addressed this in 8654c4a.